clean_repository

```diff
@@ -1,47 +1,30 @@
 #!/bin/sh
 
-mivar=`du -a | grep cmake_install.cmake | awk '{print $2}'`
+set -e
 
-mv ./doc/paper-autonomous_robots/Makefile /tmp
+test -d src/stable && test -d src/testing
 
-for file in $mivar; do
-	rm -f $file
-done
-
-mivar=`du -a | grep Makefile$ | awk '{print $2}'`
-
-
-for file in $mivar; do
-	rm -f $file
-done
-
-mivar=`du -a | grep CMakeFiles | awk '{print $2}'`
-
-
-for file in $mivar; do
-	rm -fr $file
-done
-
-mivar=`du -a | grep CMakeCache.txt | awk '{print $2}'`
+# remove cmake_(un)?install.cmake files
+find -type f -name cmake_install.cmake -o -name cmake_uninstall.cmake | xargs -r rm
 
+# remove Makefile files
+find src -type f -name Makefile | xargs -r rm
+test -d CMakeFiles \
+    && find CMakeFiles -type f -name Makefile | xargs -r rm \
+    || true
```

Comment:
`|| true` means that this command will fail, and `set -e` will kick script. Why it fails? (I can't see it, sorry)

Is there some `Makefile` that must be retained?